src/Vocabularies/LDP.ts

```diff
@@ -45,6 +46,7 @@ export const LDP:{
 	Container:"http://www.w3.org/ns/ldp#Container";
 	Descending:"http://www.w3.org/ns/ldp#Descending";
 	DirectContainer:"http://www.w3.org/ns/ldp#DirectContainer";
+	ExecutableQuery: "http://www.w3.org/ns/ldp#ExecutableQuery";
```

Comment:
This should be part of the C namespace, not in the LDP namespace. This same change has also been requested in the platform implementation. You can confirm this change with the platform team.